src/adaptors/aave-v3/index.js

```diff
@@ -354,7 +356,10 @@ const apy = async () => {
     return chainPools;
   });
 
-  const ethPools = await ethV3Pools();
+  // This results in a call exception
+  const ethPools = await ethV3Pools().catch((err) => {
```

Comment:
pls revert that change, i specifically want to break the adapter if ethV3Pools fails